### ISSUE TITLE
Render escape games and games shops

### ISSUE THREAD

**Issue by dawagner** [gravitystorm/openstreetmap-carto#3368]
I suggest adding icons for leisure=escape_game, shop=games and possibly other. It seems reasonable that they would share a same icon considering that they are not so frequently used yet.

I'd find the icon of a 6-sided dice appropriate.

**Comment by dieterdreist** [comment]
sent from a phone

> On 29. Aug 2018, at 23:31, David Wagner <notifications@github.com> wrote:
> 
> I suggest adding icons for leisure=escape_game, shop=games and possibly other


for now it is only other that meets the requirements, because the first is currently used about 300 times and the second 600 times, while a rough limit of this style is more than 2000 of something in order to consider adding it. It is a general style.

**Comment by kocio-pl** [comment]
Both are raising, so it's probably only a matter of time. However we need a real icon too and we can start with designing it now.

![taghistory 30](https://user-images.githubusercontent.com/5439713/44847302-5568e180-ac53-11e8-863d-3cabeca84e05.png)

**Comment by Tomasz-W** [comment]
6-sided dice icon:

![shop games](https://user-images.githubusercontent.com/25656654/45775618-9780b400-bc50-11e8-8d3c-661e332708f1.png)

It's not working for me, so I would stay with shop-vlolet generic dot for shop=games. For leisure=escape_game I propose old key shape (eg. https://thumbs.dreamstime.com/thumblarge_3545/35456913.jpg) or leisure-green generic dot.

**Comment by kocio-pl** [comment]
Too small and detailed for me. I would rather go with running man icon, probably with an arrow, but green running man is already used for `leisure=fitness_centre`.

**Comment by jeisenbe** [comment]
That looks like a 20-sided die? Good idea, but too complicated

How about a cube-shaped, six-sided die?

Or would that be too similar to gambling casinos?

**Comment by Tomasz-W** [comment]
I think icon like that might be confusing for this feature. It's called "escape game", but as I know, usually it's not about literally running away from something, but about to patiently solving some jigsaws etc. Another possible problem is that we are planning leisure-green sport icons for some of `leisure=sport_centre` + `sport=*` combinations, then this symbol would be propably wrongly considered as some dyscipline icon.

**Comment by kocio-pl** [comment]
So maybe green question mark?

**Comment by Adamant36** [comment]
@Tomasz-W, yeah but you solve the problems to get out the door and escape the room eventually. Plus, I actually got that from doing a Google search for escape games. What about any of these? https://thenounproject.com/search/?q=escape%20game I like the key and lock thing maybe.

**Comment by Tomasz-W** [comment]
I'll try with some old-looking key shape.

**Comment by meased** [comment]
A lock would be very recognizable:
![lock](https://user-images.githubusercontent.com/14190496/45919741-1fcca800-be4f-11e8-86c6-5dc3a1eddeee.png)
combined with leisure-green, I think would get the idea across.

**Comment by polarbearing** [comment]
Such lock symbol is frequently used for secure (SSL) connections. I would be puzzled to recognise games here. 
What about a keyhole then?

**Comment by dieterdreist** [comment]
are we seriously discussing an icon for a leisure feature with not even 400 uses worldwide?

sent from a phone

**Comment by Adamant36** [comment]
@dieterdreist, how many of these places do you think there even are in the world? Id imagine its a rather niche thing in the first place.

**Comment by dieterdreist** [comment]
sent from a phone

> On 22. Sep 2018, at 23:56, Adamant36 <notifications@github.com> wrote:
> 
> @dieterdreist, how many of these places do you think there even are in the world? Id imagine its a rather niche thing in the first place.
> 


exactly, and the same amount of people will be interested in them. I would not include them for the moment, maybe later if it really becomes a thing

**Comment by Tomasz-W** [comment]
That's also the reason why I'm still proposing introducing _leisure-green_ generic dot. At the one hand it may be useful to somebody to find escape game place on map, but on the other I can understand an argument that special icon for something with 400 uses would be inappropriate. Similar problem with some `shop=*` rare tags was resolved this way, and it works good.

**Comment by Adamant36** [comment]
At least a dot would mean the names of the places would be rendered. I think that should be the baseline. Otherwise, people aren't going to know by looking at the map if the place should be added or not.

**Comment by kocio-pl** [comment]
Green dot for leisure makes sense for me. Amenity is too wide to use brown dots, but we have dots for shops and offices already and I've just proposed orange dot for gastronomy (see #3410).

Maybe there are also some other leisure objects for which we might also use green dot?

**Comment by Tomasz-W** [comment]
> Maybe there are some other leisure objects for which we might also use green dot?

@kocio-pl Related to #2930

**Comment by dawagner** [comment]
A generic icon (dot) + the name for uncommon leisure tags sounds fine to me.

**Comment by kocio-pl** [comment]
Would you like to try making the code then? We have still too few coders active, so it's worth trying!

**Comment by IVAN-RSM** [comment]
And lasser tag?

**Comment by kocio-pl** [comment]
I don't understand, what do you mean?

**Comment by Raphael124** [comment]
Any news on this ? (Sorry to revive some old discussion, but I've mapped a few escape games only to see them deleted because “they already existed”)

**Comment by mutipg** [comment]
The number of uses of this key is constantly growing, so rendering it on the map seems important.

https://taghistory.raifer.tech/#***/leisure/escape_game
![taghistory (1)](https://github.com/user-attachments/assets/6bc16851-4c63-4856-838c-27d280d743ce)

**Comment by Chepycou** [comment]
On that note, I wonder it there should not be a bit broader of a category, so that it may include quizz rooms or puzzle rooms.

**Comment by dfandrich** [comment]
Feel free to critique the proposed icon, but this PR will at least render something.

**Comment by mutipg** [comment]
@dfandrich, I really like your icon proposal.

**Comment by Chepycou** [comment]
Although I am not sure about the icon, I'd say it is always best to have something rather than nothing, so let's go with that one and refinement may come later on.

**Comment by Koreller** [comment]
Today in osm-carto ``shop=games`` render like this :

![Image](https://github.com/user-attachments/assets/cb79b922-0111-43d9-9b0b-c18a01a2202b)

I would to propose this one instead :

<img width="16" height="16" alt="Image" src="https://github.com/user-attachments/assets/0154fdc9-66f8-4a64-987c-f6ba63900818" />

**Comment by HolgerJeromin** [comment]
> I would to propose this one instead

~~I nearly never used a dice in an escape game.~~

Sorry, I was confused. I miss read that this icon proposal was about the shop and not the game.
The mixture in this issue is bad.

**Comment by dieterdreist** [comment]
but the icon looks good and a die is very recognizable and a good symbol for game.

**Comment by HolgerJeromin** [comment]
~~Thats why they are used for [casinos](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dcasino)~~

**Comment by Koreller** [comment]
I've create a new issue : https://github.com/openstreetmap-carto/openstreetmap-carto/issues/5199, following the [HolgerJeromin comment](https://github.com/openstreetmap-carto/openstreetmap-carto/issues/3368#issuecomment-4150530931)

**Comment by imagico** [comment]
Ok, #5199 separates out `shop=games` from `leisure=escape_game` here.
